sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -709,10 +708,11 @@ def release_key(self, name, target_attestation_token, version=None, **kwargs):
         :rtype: ~azure.keyvault.keys.ReleaseKeyResult
         :raises: :class:`~azure.core.exceptions.HttpResponseError`
         """
+        version = kwargs.pop("version", "")
         result = self._client.release(
             vault_base_url=self._vault_url,
             key_name=name,
-            key_version=version or "",
+            key_version=version,
```

Comment:
Does the release function choose the latest version when the version is empty as I don't find it to take any action here?

Reply:
Yes -- I think it's a swagger quirk that `key_version` is a required parameter for a couple of functions. Key Vault will use the default version when the version isn't specified, but because the parameter is required, we need to send an empty string instead of `None`.

Reply:
its because there's no great way to model optional path params (which key_version ends up being). So they are "required" in swagger but optional in reality. At least that's my understanding of the reasoning why...